test/validate/linux_validate.go

```diff
@@ -64,6 +72,10 @@ type Address struct {
 	Addr string `json:"ipv4"`
 }
 
+type LinuxValidator struct {
+	validator linuxValidator
```

Comment:
why have a LinuxValidator and a WindowsValidator that just hold validators? I'm guessing you are trying to implement some os-specific code. I would suggest to just remove this concept of having OS-specific validators, so just get rid of LinuxValidator and WindowsValidator, and just have one (exported) Validator. Then the os-specific branching can happen in the funcs where it needs to.

Reply:
Yeah that made sense. Removed os specifci validator.  Branching the funcs if needed based on os.